src/proof/nova.rs

```diff
@@ -241,110 +241,104 @@ impl<F: LurkField> Nova<F> for NovaProver<F> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::eval::{empty_sym_env, Status};
+    use crate::eval::empty_sym_env;
     use crate::proof::{verify_sequential_css, SequentialCS};
-    use crate::writer::Write;
+    use crate::store::ContPtr;
 
     use bellperson::util_cs::{metric_cs::MetricCS, Comparable, Delta};
     use pallas::Scalar as Fr;
 
-    const DEFAULT_CHECK_NOVA: bool = false;
```

Comment:
This default was previously preventing the tests from generating real Nova proofs for every test. I think you've removed the ability to set that (Nova proof or not) on a per-test basis and so are generating full Nova proofs for every test. I think that's also why CI is now timing out.

Reply:
This change was reverted.